pkgs/applications/video/clapper/default.nix

```diff
@@ -58,20 +54,19 @@ stdenv.mkDerivation rec {
     gtk4
     libGL
     libadwaita
-    libsoup
+    libsoup_3
     wayland
     wayland-protocols
+    vala
```

Comment:
Should go to `nativeBuildInputs`

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/306548/commits/2fd6717b51137e2ab74b11cb4117332e3f69e4ad